LBH.AdultSocialCare.Api/V1/Gateways/Common/Concrete/CareChargesGateway.cs

```diff
@@ -47,5 +53,49 @@ public async Task<ProvisionalCareChargeAmountPlainDomain> GetUsingServiceUserIdA
 
             return provisionalAmount?.ToDomain();
         }
+
+        public async Task<PagedList<CareChargePackagesDomain>> GetCareChargePackages(CareChargePackagesParameters parameters)
+        {
+            var careChargePackagesCount = await GetCareChargePackagesCount(parameters);
+            var careChargePackagesList = await GetCareChargePackagesList(parameters);
+
+            var paginatedCareChargePackageList = careChargePackagesList
+                .Skip((parameters.PageNumber - 1) * parameters.PageSize)
+                .Take(parameters.PageSize);
+
+            return PagedList<CareChargePackagesDomain>.ToPagedList(paginatedCareChargePackageList, careChargePackagesCount, parameters.PageNumber, parameters.PageSize);
+        }
+
+        private async Task<int> GetCareChargePackagesCount(CareChargePackagesParameters parameters)
+        {
+            return await _dbContext.CarePackages
+                .FilterCareChargeCarePackageList(parameters.Status, parameters.ModifiedBy, parameters.OrderByDate)
+                .CountAsync();
+        }
+
+        private async Task<List<CareChargePackagesDomain>> GetCareChargePackagesList(CareChargePackagesParameters parameters)
+        {
+            return await _dbContext.CarePackages
+                .FilterCareChargeCarePackageList(parameters.Status, parameters.ModifiedBy, parameters.OrderByDate)
```

Comment:
Furkan, 

Could you please add a filter for IsS117ClientConfirmed? You don't need to return S117Client, if it is approved. 

Btw, it can be handled with another user-story but I think, it will be easy to add now :) It is totally your decision :)